src/core/Notification/Notification.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { Component, forwardRef, ReactNode } from 'react';
+
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import {
+  HtmlDiv,
+  HtmlDivWithRef,
+  HtmlButton,
+  HtmlButtonProps,
+  HtmlDivWithRefProps,
+} from '../../reset';
+import { hLevels } from '../../reset/HtmlH/HtmlH';
+import { Icon } from '../Icon/Icon';
+import { Heading } from '../Heading/Heading';
+
+import { AutoId } from '../utils/AutoId/AutoId';
+import { getConditionalAriaProp } from '../../utils/aria';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
+import { baseStyles } from './Notification.baseStyles';
+
+export const baseClassName = 'fi-notification';
+
+export const notificationClassNames = {
+  styleWrapper: `${baseClassName}_style-wrapper`,
+  content: `${baseClassName}_content`,
+  label: `${baseClassName}_label`,
+  textContentWrapper: `${baseClassName}_text-content-wrapper`,
+  icon: `${baseClassName}_icon`,
+  iconWrapper: `${baseClassName}_icon-wrapper`,
+  closeButton: `${baseClassName}_close-button`,
+  closeButtonWrapper: `${baseClassName}_close-button-wrapper`,
+  smallScreen: `${baseClassName}--small-screen`,
+  actionElementWrapper: `${baseClassName}_action-element-wrapper`,
+};
+
+export interface NotificationProps extends HtmlDivWithRefProps {
+  /** Style variant. Affects color and icon used.
+   * @default 'neutral'
+   */
+  status?: 'neutral' | 'success' | 'error';
+  /** Set aria-live mode for the Notification text content and label.
+   * @default 'polite'
+   */
+  ariaLiveMode?: 'polite' | 'assertive' | 'off';
+  /** Label for the Notification */
+  labelText?: string;
+  /** Main content of the Notification */
+  children?: ReactNode;
+  /** Elements to be rendered under the notification text. Can be e.g. buttons, links etc. */
+  actionElements?: ReactNode;
+  /** Header variant for Notification */
+  headingVariant?: hLevels;
```

Comment:
As we are giving default value later on, we need to communicate in the API. Look example how it is done for `ariaLiveMode` prop.

Reply:
Fixed. added @default and exluded h1 from hLevels.

Reply:
Good job for excluding the h1!